app/src/main/java/com/example/anfcodetest/CustomViewHolder.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.anfcodetest
+
+import android.content.Intent
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import android.view.View
+import android.widget.Button
+import androidx.core.content.ContextCompat.startActivity
+import androidx.recyclerview.widget.RecyclerView
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.card_row.view.*
+
+class CustomViewHolder(private val view: View) : RecyclerView.ViewHolder(view) {
+
+    fun bind(card: Card) = with(view) {
+        loadBackgroundImage(card.backgroundImage)
+        title.setTextAndVisibility(card.title)
+        promoMessage.setTextAndVisibility(card.promoMessage)
+        topDescription.setTextAndVisibility(card.topDescription)
+        setUpBottomDescription(card.bottomDescription)
+        topButton.setUpButton(card.content?.firstOrNull())
+        if (card.content?.size == 2) {
+            bottomButton.setUpButton(card.content.get(1))
+        }
+    }
+
+    private fun onButtonClicked(title: String?, target: String?) = with(view) {
```

Comment:
Since our click listener was duplicated between buttons - we can have a single method and pass in the values that we need at the time of assigning the click listener. This makes it much easier to read, reduces the amount of code you have, and makes it easier to test! Any time you are writing the same code twice, try to to stop and think - could this be a separate method?